scripts/generate_docs

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+bazel build //kotlin:stardoc || fail "docs did not generate"
+
+cp -f bazel-bin/kotlin/kotlin.md docs/ || fail "couldn't copy"
```

Comment:
Did you intend to include this in this PR?

Reply:
Yeah, makes it easier to update the docs.